components/Reservations/CreateReservationModal.tsx

```diff
@@ -0,0 +1,253 @@
+import { useState } from "react";
+import { addReservation } from "../../actions/calendarActions";
+
+export default function CreateReservation({ setIsShowing }) {
```

Comment:
It's good practice that the default export has the same name as the `.tsx` .
This can be simplified to `ReservationModal` .. avoid using action words like `Create` in components.